ui/src/Styles/_colors.scss

```diff
@@ -67,8 +67,8 @@ $button-blue: #056ccb;
 $button-hover-blue: #058ceb;
 $button-active-blue: #0550a0;
 
-$background-blue: #34495e;
-$background-gray: #ecf0f1;
+$background-blue: $asphalt;
+$background-gray: $gray;
```

Comment:
It might be better if we just replace calls to background-blue or background-gray with asphalt or gray rather than having two names for the same color.

Reply:
I'm just gonna revert these lines and put in a follow up PR cleaning up the colors file.